pkg/datapath/iptables/iptables.go

```diff
@@ -741,13 +741,7 @@ func endpointNoTrackRules(prog string, cmd string, IP string, port *lb.L4Addr, i
 	return nil
 }
 
-func InstallEndpointNoTrackRules(IP string, port uint16, ipv6 bool) error {
```

Comment:
No need to rename this function IMO, just removing the
```go
	if skipPodTrafficConntrack(ipv6) {
		return nil
	}
```
check below (ditto for the `RemoveEndpointNoTrackRules` function) should suffice

Reply:
Actually if possible I would like to reverse this back to its previous name "InstallNoTrackRules". Changing its name in the first place actually broke us since we have internal patches that calls out to these functions. Thoughts?

Reply:
> Actually if possible I would like to revert this back to its previous name "InstallNoTrackRules". Changing its name in the first place actually broke us (google) since we have internal patches that calls out to these functions. When this was implemented node-local-dns was the sole use case of this feature, has that changed? I searched the code and it seems like it is still the case, so I think it won't introduce new regressions if we change these names back but maybe I missed something? Thoughts?

Reply:
Discussed offline over slack with @jibi , current logic would cover required rule regardless of the flag being set or not. I will repurpose this PR to revert the original function name and add comments to avoid confusion for readers.

Reply:
Done.